### PR TITLE
Feature/keep transcriptions

### PR DESCRIPTION
**Pull request by GuyDaniels**
Changes core to optionally (based on signal files) write out either or both audio files and text transcription files which are both written to /var/log/mycroft.  Default is set to 'off'.
Note: Associated with skill-keep-transcriptions which provides intents to start/stop the audio/text transcriptions.

### CODE REVIEW

**Comment by JarbasAI** [comment]
how exactly would this be turned on and off? i like the idea :)

**Comment by GuyDaniels** [comment]
JarbasAI : The skill-keep-transcriptions turns the file signals off/on.  It's on a skills pull request - if I did that correctly..

**Comment by JarbasAI** [comment]
couldnt Transcribe use @classmethod to avoid having to instantiate it? it doesnt really need self if i got it correctly

File: mycroft/client/speech/transcribe.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+class Transcribe:
+    """
+    Name: Transcribe
+    Purpose: Writes the transcription file.
+    Imports data to the new, more precise text file, containing
+             the lines needed.
+    """
+    @classmethod
+    def write_transcribed_files(self, audio, text):
```
Comment (forslund) [inline mycroft/client/speech/transcribe.py]: Does this have to be in a class? The self reference isn't used.

Comment (GuyDaniels) [reply mycroft/client/speech/transcribe.py]: No, it doesn't need to be in a class.  Do you think I should just move write_transcribed_files() over into listener.py?  That works for me...

**Pull request closed without merging**